c2pa/claim-examples/claim_create.json

```diff
@@ -0,0 +1,98 @@
+{
+    "vendor": "starlinglab",
+    "recorder": "Starling Capture",
+    "assertions": [
+        {
+            "label": "stds.schema-org.CreativeWork",
+            "data": {
+                "@context": "https://schema.org",
+                "@type": "CreativeWork",
+                "author": [
+                    {
+                        "@type": "Person",
+                        "credential": [
+                        ],
+                        "identifier": "https://hypha.coop",
+                        "name": "Benedict Lau"
```

Comment:
I don't believe `name` is a valid field of CreativeWork

Reply:
I took this block from the claim-tool example:
```
        {
            "label": "stds.schema-org.CreativeWork",
            "data": {
                "@context": "https://schema.org",
                "@type": "CreativeWork",
                "author": [
                    {
                        "@type": "Person",
                        "credential": [
                            {
                                "url": "did:adobe:f78db44b3d758bbf1ac2b1da23d6a9bc8d4554bbc7ca6f78f5536d6cf813d218e?service=VerifiableCredentials"
                            }
                        ],
                        "identifier": "https://cai-identity.adobe.io/identities/did:adobe:f78db44b3d758bbf1ac2b1da23d6a9bc8d4554bbc7ca6f78f5536d6cf813d218e?service=VerifiableCredentials",
                        "name": "Gavin Peacock"
                    }
                ]
            }
        }
```

I think `author` in [CreativeWork](https://schema.org/CreativeWork) is typed `Person` or `Organization`, so they are getting `name` from [Thing](https://schema.org/Thing)? I am not sure if this is how I should read the schemas though.

I am also not sure where `credential` is coming from in [`person`](https://schema.org/Person).

Reply:
We use the `name` field with our current tools and the verify site.  And yes, `name `is allowed under `Thing` from Person.

Reply:
We use `credential` as it is shown in the c2pa spec. I do not know how it maps to `CreativeWork` as defined by schema.org.

Reply:
Ok so let's leave this as is for now. I will add copyright.